README.org

```diff
@@ -227,7 +227,23 @@ Here's a screenshot using this configuration, which removes the links indicator,
 #+name: fig-indicators
 [[images/indicators.png]]
 
-You can create your own indicators, of course, though keep in mind the included predicate functions must be performance-optimized, since the completion UI runs them on your entire library every time you open it.
+You can create your own indicators, of course.
+Here's an example indicator definition incorporating icons:
+
+#+begin_src emacs-lisp
+(defvar citar-indicator-notes-icons
+  (citar-indicator-create
+   :symbol (all-the-icons-material
+            "speaker_notes"
+            :face 'all-the-icons-blue
+            :v-adjust -0.3)
+   :function #'citar-has-notes
+   :symbolwidth 2
+   :emptysymbol "  "
```

Comment:
For me, `emptysymbol` needs to be set to just one space to make it look right.

Reply:
I remove the line in the README.

Reply:
Given that it makes no difference for icons or plain text, maybe we can remove that entirely?

Reply:
Yes, I can remove it. 
But in general, I wouldn't remove this option. Having a dim placeholder icon for whenever there is no note, pdf or link seems like a nice option to have.

Reply:
Right; I meant in this case, not generally.

So I removed them from the examples in the docs and wiki page.